Source/SwiftLintFramework/Rules/ShorthandOperatorRule.swift

```diff
@@ -42,32 +43,43 @@ public struct ShorthandOperatorRule: ConfigurationProviderRule {
                 "↓foo.aProperty = foo.aProperty \(operation) 1\n",
                 "↓self.aProperty = self.aProperty \(operation) 1\n"
             ]
-
-        }
+        } + [
+            "n = n + i / outputLength",
+            "n = n - i / outputLength"
+//            "d = d * 60 * 60"
+        ]
     )
 
     private static let allOperators = ["-", "/", "+", "*"]
 
-    private static let pattern: String = {
+    private static let patterns: [String] = {
         let escaped = { (operators: [String]) -> String in
             return "[\(operators.map { "\\\($0)" }.joined())]"
         }
 
-        let escapedOperators = escaped(allOperators)
+        let escapedAll = escaped(allOperators)
+        let operatorsWithoutPrecedence = escaped(["-", "+"])
+        let operatorsWithPrecedence = escaped(["/", "*"])
         let operand = "[\\w\\d\\.]+?"
         let spaces = "[^\\S\\r\\n]*?"
 
-        let pattern = "^\(spaces)(\(operand))\(spaces)=\(spaces)(\\1)\(spaces)\(escapedOperators)"
-        return pattern
+        let pattern1 = "^\(spaces)(\(operand))\(spaces)=\(spaces)(\\1)\(spaces)\(operatorsWithoutPrecedence)"
+        let pattern2 = "^\(spaces)(\(operand))\(spaces)=\(spaces)(\\1)\(spaces)\(operatorsWithPrecedence)\(spaces)\\S+$"
+        return [pattern1, pattern2]
```

Comment:
could this be a single regex composed by `pattern1` and `pattern2` with an *or*?

This way the rule would be a bit faster.